frontend/apps/app/vercel.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://openapi.vercel.sh/vercel.json",
+  "buildCommand": "cd ../../../ && pnpm build --force --filter @liam-hq/app",
+  "installCommand": "pnpm install",
+  "devCommand": "next",
+  "framework": "nextjs",
+  "functions": {
+    "app/**/*": {
+      "supportsCancellation": true
+    }
```

Comment:
https://vercel.com/docs/functions/functions-api-reference#cancel-requests

Without this setting, AbortSignal could not be used on Vercel.